docs/preview/connected-registry/troubleshooting.md

```diff
@@ -26,3 +24,88 @@ May include one or more of the following:
 ### Configure Docker daemon to access insecure registry
 
 The access the connected registry via HTTP, you must configure the client Docker daemon to allow access to insecure registries. The steps are described in [Test an insecure registry](https://docs.docker.com/registry/insecure/) article on Docker's web site.
+
+## Symptoms
+
+* Unable to pull an image from the connected registry. Client error is
+`Error response from daemon: manifest for <connected-registry-login-server-ip-or-dns>/<repository-name>:<tag> not found: manifest unknown: manifest unknown`
+
+## Causes
+
+* The connected registry is not configured to sync this repository from the Azure Container Registry.
+
+## Potential solutions
+
+### Configure the connected registry to sync the repository
+
+In order to access this image, you must update the connected registry configuration to sync the repository. From the Az CLI run
+`az acr connected-registry repo -r <ACR-name> -n <connected-registry-name> --add <repository-name>`
+
+Wait a few minutes for the connected registry to sync the repository and try pulling the image again.
+
+## Symptoms
+
+* Unable to push or pull images to or from the connected registry. Client error is
+`Error response from daemon: pull access denied for <connected-registry-login-server-ip-or-dns>/<repository-name>, repository does not exist or may require 'docker login': denied: Insufficient scopes to perform the operation`
+
+## Causes
+
+* The repository is synced to the connected registry, but the client token used for `docker login` does not have access.
+
+## Potential solutions
+
+### Assign permissions to the connected registry client token
+
+To update the permissions of the client token, you must update the corresponding scope map. To view the scope map resource ID associated with a token, run the following on the Az CLI:
+
+`az acr token show -r <ACR-name> -n <client-token-name> -o tsv --query scopeMapId`
+
+#### Pull permissions
+
+To give the client token pull permissions to the repository, run the following from the Az CLI:
+
+```
+az acr scope-map update \
+  --name <scope-map-name> \
+  --registry <ACR-name> \
+  --add-repository <repository-name> content/read
+```
+
+#### Push permissions
+
+To give the client token push permissions to the repository, run the following from the Az CLI:
```

Comment:
Can we add a note that this is only applicable for connected registry working in `registry` mode?